src/server/sv_main.c

```diff
@@ -339,6 +339,12 @@ SV_RunGameFrame(void)
 	{
 		time_before_game = Sys_Milliseconds();
 	}
+	/*{
+		static int gametime = 0;
+		int newtime = Sys_Milliseconds();
+		printf("since last gameframe: %d ms\n", newtime - gametime);
+		gametime = newtime;
+	}*/
```

Comment:
Shouldn't this be removed? Or commented in and hidden behind something like `developer`?

Reply:
oh, yes, I didn't mean to commit that